src/Microsoft.DncEng.SecretManager/Commands/SynchronizeCommand.cs

```diff
@@ -48,8 +63,14 @@ public override async Task RunAsync(CancellationToken cancellationToken)
 
                 bool regenerate = false;
 
-                if (existing.Any(e => e == null))
+                if (_force)
+                {
+                    _console.WriteLine("-force is set, will rotate.");
```

Comment:
I feel like this definitely needs a confirm, since this will break stuff, yes?

Reply:
it won't actually break anything unless you run this twice quickly, the sas tokens have a 30 day expiry, and the connection strings all have 2 keys that get rotated separately so the n-1 string still works. I can add a confirm though. This should only be done by a human so its fine to have a confirm here.